public/wallet-service-worker.mjs

```
@@ -6465,7 +6469,11 @@ function Ro(t) {
     createdAt: new Date(t.createdAt * 1e3)
   };
 }
-function Kh({ connectorInput: t, vtxoInput: e, vtxoAmount: n, connectorAmount: r, feeAmount: i, vtxoPkScript: a, connectorPkScript: c, serverPkScript: f, txLocktime: d }) {
+function Kh(t) {
+  const e = (n) => n instanceof Error && (n.name === "HeadersTimeoutError" || n.name === "BodyTimeoutError" || n.code === "UND_ERR_HEADERS_TIMEOUT" || n.code === "UND_ERR_BODY_TIMEOUT");
```

ESLint

[error] Strings must use singlequote.

(@typescript-eslint/quotes)